ql/src/test/queries/clientpositive/test_unixtimestamp_for_invaild_dates.q

```diff
@@ -0,0 +1,20 @@
+DESCRIBE FUNCTION UNIX_TIMESTAMP;
```

Comment:
Instead of qtests better opt for unit tests. They are much faster and in this case they offer the exact same code coverage. Please add any new tests under `TestGenericUDFToUnixTimestampEvaluateStringString` and include also the output and result for `SIMPLE` formatter.

IF the results between SIMPLE and DATETIME formatter are different explain why and clarify if it is desired.

Reply:
Sure, i have added a new test in TestGenericUDFToUnixTimestampEvaluateStringString for GenericUDFUnixTimestamp. There are a few changes in the results between SIMPLE and DATETIME  . I will add update the PR description detailing all the changes